include/cantera/thermo/ThermoPhase.h

```diff
@@ -563,9 +563,9 @@ class ThermoPhase : public Phase
         throw NotImplementedError("ThermoPhase::getPartialMolarVolumes");
     }
 
-    //! @}
-    //! @name Properties of the Standard State of the Species in the Solution
-    //! @{
+    //@}
+    /// @name Properties of the Standard State of the Species in the Solution
+    //@{
```

Comment:
This change should be reverted - the lines containing `@{` and `@}` need to be Doxygen comment lines (starting with `//!` or `///`, as they were.